docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -59,26 +76,20 @@ cpu `requests` also run on CPUs in the shared pool. Only containers that are
 both part of a `Guaranteed` pod and have integer CPU `requests` are assigned
 exclusive CPUs.
 
-**Note:** When reserving CPU with `--kube-reserved` or `--system-reserved` options, it is advised to use *integer* CPU quantities.
+**Note:** The kubelet requires `kubereserved.cpu` plus `systemreserved.cpu` to
```

Comment:
I haven't seen the kube/system CPU reservation expressed like `kubereserved.cpu` before.  Should we just say:

The kubelet requires a CPU reservation greater than zero be made using either --kube-reserved and/or --system-reserved when the static policy is enabled."

Reply:
Will change

Reply:
Fixed